demo-extended/templates/deployment.yaml

```diff
@@ -31,18 +34,38 @@ spec:
           {{- end }}
           {{- end }}
           {{- if or ((.Values.scaling).resources).memory ((.Values.scaling).resources).cpu }}
+
           resources:
-            {{- if ((.Values.scaling).resources).memory }}
```

Comment:
Can you rever the resources object since its not format properly and it can cause a null pointer because of deleted brackets

Reply:
ok